stable/weblate/values.yaml

```diff
@@ -79,14 +79,17 @@ service:
   port: 80
 
 ingress:
-  enabled: false
+  enabled: true
```

Comment:
Why is the default switched from `false` to `true`?

Reply:
it was changed to test new ingress changes in ci checks as false will neglect creating ing resources

Reply:
reverted after testing